app/helpers/settings_helper.rb

```diff
@@ -20,7 +20,7 @@ def show_value(setting)
     when "array"
       "[ " + setting.value.join(", ") + " ]"
```

Comment:
Do we have arrays of encrypted values or is this something we need to care about?

Reply:
I don't recall seeing any. What would be the right solution then? Replace the elements with asterisks or the entire array to hide the number of elemenets?

Reply:
I don't recall any either, so we can deal with this when we get there?

Reply:
Sounds good to me